spaceship/lib/spaceship/test_flight/build.rb

```diff
@@ -0,0 +1,172 @@
+require 'time'
+
+module Spaceship::TestFlight
+  class Build < Base
+    # @example
+    #   "com.sample.app"
+    attr_accessor :bundle_id
+
+    # @example
+    #   "testflight.build.state.testing.active"
+    # @example
+    #   "testflight.build.state.processing"
+    attr_accessor :internal_state
+
+    # @example
+    #   "testflight.build.state.submit.ready"
+    # @example
+    #   "testflight.build.state.processing"
+    attr_accessor :external_state
+
+    # Internal build ID (int)
+    # @example
+    #   19285309
+    attr_accessor :id
+
+    # @example
+    #   "1.0"
+    attr_accessor :train_version
+
+    # @example
+    #   "152"
+    attr_accessor :build_version
+
+    attr_accessor :beta_review_info
+
+    attr_accessor :export_compliance
+
+    attr_accessor :test_info
+
+    attr_accessor :install_count
+    attr_accessor :invite_count
+    attr_accessor :crash_count
+
+    attr_accessor :did_notify
```

Comment:
This is never used - do we want to create attr_accessor's for things we don't access?